docs/source/data/subscriptions.mdx

```diff
@@ -540,6 +548,17 @@ The following are community-created `PubSub` libraries for popular event-publish
 
 If none of these libraries fits your use case, you can also create your own `PubSubEngine` subclass. If you create a new open-source library, click **Edit on GitHub** to let us know about it!
 
-## The `graphql-ws` transport library
+## Old `subscriptions-transport-ws` library
+
+>If use subscriptions with Apollo Client you must ensure **[both your client and server subscription protocols are consistent](https://www.apollographql.com/docs/react/data/subscriptions/#choice-of-subscription-protocol)** for the subscription library you are using.
+
+This article details how to setup the [`graphql-ws`](https://www.npmjs.com/package/graphql-ws) library to add support for subscriptions to Apollo Server. Previously this article recommended using the `subscriptions-transport-ws` library, however this library is no longer actively maintained.
+
+Apollo Server supports both `graphql-ws` and `subscriptions-transport-ws`, but we strongly recommend using the currently maintained `graphql-ws` library.
+
+Both `graphql-ws` and `subscriptions-transport-ws` implement different protocols for GraphQL subscriptions over websockets. Meaning, if you do decide to switch from one library to another you will need to ensure your [Apollo Client subscription protocol](https://www.apollographql.com/docs/react/data/subscriptions/#choice-of-subscription-protocol) is **consistent** with the subscription library you are using within Apollo Server and vice versa.
+
+Note that you might need to [adjust the Apollo Sandbox settings to use the same subscription protocol as the subscription library you are using](https://www.apollographql.com/docs/studio/explorer/additional-features/#subscription-support).
```

Comment:
Also I'd suggest replacing this with a brief section (formatted however makes sense, bulleted list?) explaining how to use graphql-ws with each part of the platform:
- settings for Explorer
- AC web: use (not yet released) AC 3.6 (https://github.com/apollographql/apollo-client/pull/9067/files) and GraphQLWsLink instead of WebSocketLink (https://www.apollographql.com/docs/react/api/link/apollo-link-subscriptions/ will be the doc link once released; here's a preview: https://github.com/apollographql/apollo-client/pull/9067/files)
- AC Kotlin: https://www.apollographql.com/docs/kotlin/essentials/subscriptions/#customizing-your-websocket-protocol
- AC iOS: not yet supported